web/static/as_printable_html/as_printable_html.js

```diff
@@ -208,64 +188,59 @@ annotationRanges.forEach((rg) => {
       // Transparently replace the content inside the node
       ranges.forEach((range) => {
         const deletion = document.createElement("del");
-        deletion.setAttribute('correction-deletion-id', id);
+        deletion.setAttribute("correction-deletion-id", id);
         deletion.setAttribute("datetime", datetime);
         deletion.classList.add(type);
 
-        const replacement = document.createElement("ins");
-        replacement.setAttribute('correction-insertion-id', id);
-        replacement.setAttribute("datetime", datetime);
-        replacement.classList.add(type);
-        replacement.innerText = content;
+        const replacement = `<ins data-${type}-insertion-id="${id}" datetime="${datetime}" class="${type}">${content}</ins>`;
         range.surroundContents(deletion);
-        deletion.insertAdjacentElement("afterend", replacement);
+        deletion.insertAdjacentHTML("afterend", replacement);
       });
       break;
-
     }
     case "link":
       // Inject a hyperlink which in print will be styled as a footnote
       ranges.forEach((range) => {
         const anchor = document.createElement("a");
         anchor.setAttribute("href", content);
         range.surroundContents(anchor);
-        hyperlinkFootnote(anchor, content, new Date(datetime))
+        hyperlinkFootnote(anchor, content, new Date(datetime));
       });
-      break
+      break;
   }
 });
 
 // Any existing URLs that are external and don't open in a new tab should
-tmpl.content.querySelectorAll('a[href^="http"]:not([target])').forEach((el) => {
-  el.setAttribute('target', '_blank');
+main.querySelectorAll('a[href^="http"]:not([target])').forEach((el) => {
```

Comment:
Potential trade-off to be made here: If you're solely focusing on time to interactive, a broad event delegation filtering clicks on `a` could might work here.

Reply:
Oh that's a really good point, especially since most of these will not be clicked and there can be hundreds.